packages/docs/package.json

```diff
@@ -70,10 +72,10 @@
   },
   "lint-staged": {
     "*.{js,ts}": [
-      "vue-cli-service lint --fix"
+      "vue-cli-service lint"
```

Comment:
Why get rid of "--fix"es?

Reply:
because this linter has no `--fix` parameter, it starts formatting by default.  
There is only a `--no-fix` parameter  
https://cli.vuejs.org/core-plugins/eslint.html#injected-commands